### ISSUE TITLE
Recommendations for SNP array or WGS

### ISSUE THREAD

**Issue by daccachejoe** [wheaton5/souporcell#197]
Hi-

I am looking to run 10X Multiome (sc paired RNA & ATAC) and I want to multiplex my samples and use a SNP array or WGS to demultiplex using SouporCell. I have not run the experiment yet so I was wondering if you could offer insight as to which to use (SNP vs WGS), what depth to sequence for the genomic information, what SNP array to use if thats the method?

Thanks!

**Comment by wheaton5** [comment]
The whole point of souporcell over demuxlet is that you dont need the snp array or wgs data. It demultiplexes from the scrnaseq alone.

**Comment by jad362** [comment]
Right... Is the performance drastically improved with it? sorry for the questions, just trying to get my bearings

**Comment by wheaton5** [comment]
Is the question “does the performance drastically improve with the wgs/snp chip data” or what? If so, for >8 individuals multiplexed, you will see performance improvements with the additional data. And you will have the information about which cluster is which individual. For the clustering performance below 8 individuals, its probably better to increase the sequencing on the scrnaseq data rather than providing the wgs data. More umi/cell gives less sparse data allowing for better clustering. You can also recover “which cluster is which individual” with overlapping mixtures as noted in our paper (log2(# individuals+1) mixture experiments to do this).

**Comment by jad362** [comment]
Yes that was my question, thanks Haynes! We will like to know which individual is which so having the genomic information will be helpful. Any specifications or will a typical snp array of top represented snps be enough?

**Comment by jad362** [comment]
Thanks for your help!
